goss-testing/tests/ncn/goss-cray-tftp-check.yaml

```diff
@@ -0,0 +1,49 @@
+#
+# MIT License
+#
+# (C) Copyright 2023 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+
+{{ $scripts := .Env.GOSS_BASE | printf "%s/scripts" }}
+{{ $logrun := $scripts | printf "%s/log_run.sh" }}
+{{ $cmsdev := "/usr/local/bin/cmsdev" }}
+command:
+    {{ $testlabel := "cray_tftp_check" }}
+    {{$testlabel}}:
+        title: cray-tftp service is healthy
+        meta:
+            desc: If this test fails, run `{{$cmsdev}} test tftp` on the target node for more information
+            sev: 0
+        # Run test in verbose mode, since we're capturing the output
+        exec: |-            
+            "{{$logrun}}" -l "{{$testlabel}}" \
+                "{{$cmsdev}}" test -v tftp
+        exit-status: 0
+        stdout:
+            - SUCCESS
+        # 3 minute test timeout
+        timeout: 180000
+        # skip this test on vshasta
+        {{ if eq true .Vars.vshasta }}
```

Comment:
Do we know this won't work on vshasta?   This seems like something that should work there.

Reply:
Ok, will do.

Reply:
https://jira-pro.its.hpecorp.net:8443/browse/CASMCMS-8475